### DISCUSSION TITLE
Can NOT connecor elasticsearch(run at k8s) with trino

### DISCUSSION THREAD

**Discussion by samzong** [trinodb/trino#14740] [Q&A]
Hi guys !
I got error with use trino connector with elasticsearch.

I have an elasticsearch run at Kubernetes , and export service as : https://172.16.10.32:32091/
and add es.properties in trino is cannot working

> this is my config
```
connector.name=elasticsearch
elasticsearch.host=172.16.10.32
elasticsearch.port=32091
elasticsearch.default-schema-name=default
elasticsearch.security=PASSWORD
elasticsearch.auth.user=elastic
elasticsearch.auth.password=
elasticsearch.tls.enabled=true
# elasticsearch.tls.keystore-path=/etc/trino/pemdata/tls.key.pem
# elasticsearch.tls.truststore-path=/etc/trino/pemdata/tls.crt.pem
elasticsearch.ignore-publish-address=true
# elasticsearch.tls.verify-hostnames=false
```

and i got error like this.

![image](https://user-images.githubusercontent.com/13782141/197712186-e7c92e88-020f-433d-9faf-0c19e1529be3.png)

![image](https://user-images.githubusercontent.com/13782141/197712118-06b94fb9-ff38-441c-8e3a-b1edf1dcba1a.png)

**Comment by jbackofe** [comment]
Having a similar issue. Were you able to resolve this?
